pkg/samples/fixtures.go

```diff
@@ -0,0 +1,283 @@
+package samples
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"reflect"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/joho/godotenv"
+	"github.com/spf13/afero"
+	"github.com/thedevsaddam/gojsonq"
+
+	"github.com/stripe/stripe-cli/pkg/requests"
+)
+
+// SupportedVersions is the version number of the fixture template the CLI supports
+const SupportedVersions = 0
+
+type metaFixture struct {
+	Version int `json:"_version"`
+}
+
+type fixtureFile struct {
+	Meta     metaFixture       `json:"_meta"`
+	Fixtures []fixture         `json:"fixtures"`
+	Env      map[string]string `json:"env"`
+}
+
+type fixture struct {
+	Name   string      `json:"name"`
+	Path   string      `json:"path"`
+	Method string      `json:"method"`
+	Params interface{} `json:"params"`
+}
+
+// Fixture contains a mapping of an individual fixtures responses for querying
+type Fixture struct {
+	Fs        afero.Fs
+	APIKey    string
+	BaseURL   string
+	responses map[string]*gojsonq.JSONQ
+	fixture   fixtureFile
+}
+
+// NewFixture creates and executes a fixtures steps for populating test data
+func (fxt *Fixture) NewFixture(file string) error {
```

Comment:
I think it's conventional for functions named `NewFoo` to initialize and return a new `Foo` struct. That's not what's happening here, so I think we should change it in one of two ways:
1. rename the method to `Load` or `LoadFromFile` or something
2. keep `NewFixture`, but make it a non-struct method that actually initialize and returns a new `Fixture`

I have a slight preference for the second way, but up to you.

Reply:
Ahh yea, sorry about that! Will change that now.